src/mon/PGMap.cc

```diff
@@ -4602,16 +4607,38 @@ int reweight::by_utilization(
       osd_util.second = (double)p.second.kb_used / (double)p.second.kb;
     }
     util_by_osd.push_back(osd_util);
+    
+    // Overweighted devices and their weights are factors to
+    // calculate reweight_emergency. One 10% underfilled device
+    // with 5 2% overfilled devices, is arguably a better
+    // situation than one 10% overfilled with 5 2% underfilled devices
+    
+    if ( osd_util.second >= average_util){
+      unsigned weight = osdmap.get_weight(p.first);
+      sum_weights += weight;
+
+      // cdf of normal distribution:https://stackoverflow.com/a/18786808
+      reweight_urgency +=
+	weight * (erfc(-((osd_util.second - average_util)/average_util) * M_SQRT1_2)-1);
```

Comment:
why we can model the distribution of the deviation from average utilization using normal distribution?

the cdf of normal distribution is: \Phi(x) = 1 / 2 erfc(-x / \sqrt{2}), per https://en.wikipedia.org/wiki/Error_function#Cumulative_distribution_function

assuming the random variable in this case is `(osd_util.second - average_util)/average_util`, the cdf should be: 
```1 / 2 * erfc(-((osd_util.second - average_util) / average_util) * M_SQRT1_2)```
but not
```
weight * (erfc(-((osd_util.second - average_util)/average_util) * M_SQRT1_2)-1);
```

so, could you elaborate on how `reweight_urgency` is related to CDF of a normal distribution. and what the normal distribution is?